addons/mail/static/src/env/env.js

```diff
@@ -0,0 +1,104 @@
+odoo.define('mail/static/src/env/env.js', function (require) {
+'use strict';
+
+const ModelManager = require('mail/static/src/model/model_manager.js');
+
+const env = require('web.commonEnv');
+
+const { Store } = owl;
+const { EventBus } = owl.core;
+
+/**
+ * Messaging store
+ */
+const store = new Store({
+    env,
+    state: {},
+});
+
+/**
+ * Registry of models.
+ */
+env.models = {};
+/**
+ * Environment keys used in messaging.
+ */
+Object.assign(env, {
+    autofetchPartnerImStatus: true,
+    destroyMessaging() {
+        if (env.modelManager) {
+            env.modelManager.deleteAll();
+            env.messaging = undefined;
+        }
+    },
+    disableAnimation: false,
+    isMessagingInitialized() {
+        if (!this.messaging) {
+            return false;
+        }
+        return this.messaging.isInitialized;
+    },
+    loadingBaseDelayDuration: 400,
+    messaging: undefined,
+    messagingBus: new EventBus(),
+    modelManager: new ModelManager(),
+    store,
+});
+
+/**
+ * Components cannot use web.bus, because they cannot use
+ * EventDispatcherMixin, and webclient cannot easily access env.
+ * Communication between webclient and components by core.bus
+ * (usable by webclient) and messagingBus (usable by components), which
+ * the messaging service acts as mediator since it can easily use both
+ * kinds of buses.
```

Comment:
Indeed, forgot to make changes.
Just small cleaning that doesn't change behaviour, so not high priority.
Made task-id 2277196 so that we don't forget to make this change.